test/e2e/common/framework.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import "k8s.io/kubernetes/test/e2e/framework"
+
+// SIGNetworkDescribe annotates the test with the SIG Network label.
+func SIGNetworkDescribe(text string, body func()) bool {
+	return framework.KubeDescribe("[sig-network] "+text, body)
+}
```

Comment:
We will have to add a new one of these functions for every SIG, any reason why we don't do a generic one which can take an argument and output `[sig-{arg}]`?

Reply:
`I would say that the cost of adding 3 boiler-plate lines isn't a big cost.
OTOH - that gives compile time guarantees that if we implement that function correctly, the other also will have correct tags.

[I don't feel very strongly about this though...]